install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
What's the usage of this file? Why should it be put in the root folder?

Reply:
Shall we move this file to another location?

Reply:
How about under scripts?

Reply:
Nevermind, I guess the current path is fine. Let's merge this and not trigger CI again :)